src/ensembl/production/metadata/api/sample/ensembl_metadata/table.sql

```diff
@@ -176,7 +176,9 @@ CREATE TABLE genome
     genome_uuid varchar(128) not null,
     assembly_id int          not null,
     organism_id int          not null,
-    created     datetime(6) not null,
+    created     datetime(6)  not null,
+    is_best     tinyint(1)   not null,
```

Comment:
This should reflect the default then, right? 
```suggestion
    is_best     tinyint(1)   not null default 0,
```